var/spack/repos/builtin/packages/neurodamus-model/package.py

```diff
@@ -150,8 +183,45 @@ def setup_environment(self, spack_env, run_env):
             run_env.prepend_path('PYTHONPATH', self.prefix.python)
             run_env.set('NEURODAMUS_PYTHON', self.prefix.python)
 
+
 @contextmanager
 def profiling_wrapper_on():
     os.environ["USE_PROFILER_WRAPPER"] = "1"
     yield
     del os.environ["USE_PROFILER_WRAPPER"]
+
+
+# Aux funcs
+# ---------
+def copy_all(src, dst, copyfunc=shutil.copy):
```

Comment:
with symlinks, this wont be used at all?

Reply:
Will be, to merge with neurodamus-core. Anyway I find this a very useful little func. Maybe we could add to spack core.